### DISCUSSION TITLE
Vercel Only Deploying some commits

### DISCUSSION THREAD

**Discussion by SGJIII** [vercel/community#3826] [Help]
### Question

Vercel has only deployed 2 of my last 5 commits. It seems basically random. 

<img width="1173" alt="image" src="https://github.com/vercel/community/assets/12156051/b7269349-92b3-4d16-a384-6d5ca1ef5bd5">

**Comment by amyegan** [comment]
Hey @SGJIII. Do you have [Ignored Build Step](https://vercel.com/docs/projects/overview#ignored-build-step) configured in project settings?
